plugins/main/bridge/README.md

```diff
@@ -28,6 +28,17 @@ If the bridge is missing, the plugin will create one on first use and, if gatewa
 }
 ```
 
+## L2-only configuration
```

Comment:
I think "Example L2-only configuration" would be a better title?

Reply:
Done